addOns/openapi/src/test/resources/org/zaproxy/zap/extension/openapi/v2/PetStore_defn.json

```diff
@@ -983,8 +983,7 @@
                "$ref":"#/definitions/Category"
             },
             "name":{
-               "type":"string",
-               "example":"doggie"
+               "type":"string"
```

Comment:
Is there a problem handling this? Will it fail if a v2 defn with examples is encountered in the wild?